src/session/error.rs

```diff
@@ -180,25 +180,35 @@ impl From<MissingFieldError> for ParseError {
     }
 }
 
-/// An error type for situations where a given proto value is not supported.
+/// An error type for situations where proto enum has a value for which
+/// the definition is not known.
 #[derive(Debug)]
-pub struct UnsupportedValueError<T> {
-    /// A name of the field the value belongs to.
+pub struct UnknownEnumValueError {
```

Comment:
I think `UnsupportedValueError` and `UnknownEnumValueError` are different, I don't think one should replace another.

Reply:
Brought back UnsupportedValueError - it was removed by accident during a merge.